modules/node_modules/@ncigdc/components/SSM.js

```diff
@@ -210,23 +205,13 @@ const Mutation = compose(
             alignSelf: 'flex-start',
           }}
         />
-        {externalDbIds && !!Object.keys(externalDbIds).length &&
+        {dbSNP && /rs([0-9]+)$/g.test(dbSNP) &&
           <EntityPageVerticalTable
             title={<span><BookIcon style={{ marginRight: '1rem' }} /> External References</span>}
-            thToTd={
-              Object.keys(externalDbIds)
-              .filter(x => x !== '__dataID__')
-              .map(db => ({
-                th: db.replace(/_/g, ' '),
-                td: externalDbIds[db] && !!externalDbIds[db].length && (
-                  <ExternalLink
-                    href={externalReferenceLinks[db](externalDbIds[db][0])}
-                  >
-                    {externalDbIds[db]}
-                  </ExternalLink>
-                ),
-              }))
-          }
+            thToTd={[
+              { th: 'Name', td: 'dbSNP' },
```

Comment:
@bessierogel when you say `Name` and `value` do you mean for it to look like this 
![image](https://cloud.githubusercontent.com/assets/3953093/23914915/cbc1bf36-08bd-11e7-83e3-002b4e995224.png)
or would `dbSNP` be on the left and `none` would be on the right?